sigma/backends/panther/panther_sdyaml_backend.py

```diff
@@ -0,0 +1,307 @@
+        return {"Key": cond.field, "Condition": self.SDYAML_CONDITION_EQUALS, "Value": cond.value.to_plain()}
+
+    def convert_condition_field_eq_field(self, cond: ConditionFieldEqualsValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError()
+
+    def convert_condition_field_eq_val_str_case_sensitive(self, cond: ConditionFieldEqualsValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError()
+
+    def convert_condition_field_exists(self, cond: ConditionFieldEqualsValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError()
+
+    def convert_condition_field_not_exists(self, cond: ConditionFieldEqualsValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError()
+
+    def convert_condition_field_eq_val_bool(self, cond: ConditionFieldEqualsValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError()
+
+    def convert_condition_field_eq_val_re(self, cond: ConditionFieldEqualsValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError("Regexp is not suppoerted in sdyaml")
+
+    def convert_condition_field_eq_val_cidr(self, cond: ConditionFieldEqualsValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError()
+
+    def convert_condition_field_compare_op_val(self, cond: ConditionFieldEqualsValueExpression, state: ConversionState) -> Any:
+        raise (SigmaFeatureNotSupportedByBackendError())
+
+    def convert_condition_field_eq_val_null(self, cond: ConditionFieldEqualsValueExpression, state: ConversionState) -> Any:
+        return {"KeyPath": cond.field, "Condition": self.SDYAML_IS_NULL}
+
+    def convert_condition_field_eq_query_expr(self, cond: ConditionFieldEqualsValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError()
+
+    def convert_condition_val_str(self, cond: ConditionValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError(f'Search without specifying a Key is not supported: "{cond.value.to_plain()}".')
+
+    def convert_condition_val_num(self, cond: ConditionValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError("Enums are not supported right now")
+
+    def convert_condition_val_re(self, cond: ConditionValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError()
+
+    def convert_condition_query_expr(self, cond: ConditionValueExpression, state: ConversionState) -> Any:
+        raise SigmaFeatureNotSupportedByBackendError()
+
+    @staticmethod
+    def convert_value_str(s: SigmaString, state: ConversionState) -> str:

```

Comment:
Do we need to pass in `ConversionState` objects to the methods where we're only accessing the `SigmaString` object?

Reply:
yep, this method is defined and in base class https://github.com/SigmaHQ/pySigma/blob/main/sigma/conversion/base.py#L918
although I think `@staticmethod` should be removed